src/api/validationSchemas/commonSchemas.js

```diff
@@ -0,0 +1,59 @@
+const RootJoi = require('@hapi/joi')
+const DateExtension = require('@hapi/joi-date')
+const { Util } = require('@mojaloop/central-services-shared')
+
+const DateExtendedJoi = RootJoi.extend(DateExtension)
+const Joi = DateExtendedJoi.extend(Util.Hapi.customCurrencyCodeValidation)
+
+const guidSchema = Joi.string().guid({ version: 'uuidv4' }).label('Must be in a valid GUID format.')
+
+const dfspIdSchema = Joi.string().min(1).max(32).label('A valid DFSP number must be supplied.')
+
+const baseConditionFulfillmentSchema = Joi.string()
+  .regex(/^[A-Za-z0-9-_]{43}$/)
+  .max(48)
+  .trim()
```

Comment:
Why there is a new file for schema validation? Are we following different approach for fx schema validation?

Reply:
First of all, I think Joi shemas should have a separate place (Separation of Concerns)
And another point, that FX payloads are different from transfers one.
So I just made a bit order in schema defenitions